### ISSUE TITLE
Doesn't install on openSUSE

### ISSUE THREAD

**Issue by hrqmonteiro** [oskarsh/Yin-Yang#279]
### What happened?

```
Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
Building wheels for collected packages: systemd-python
  Building wheel for systemd-python (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [24 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-cpython-311
      creating build/lib.linux-x86_64-cpython-311/systemd
      copying systemd/__init__.py -> build/lib.linux-x86_64-cpython-311/systemd
      copying systemd/journal.py -> build/lib.linux-x86_64-cpython-311/systemd
      copying systemd/daemon.py -> build/lib.linux-x86_64-cpython-311/systemd
      creating build/lib.linux-x86_64-cpython-311/systemd/test
      copying systemd/test/test_daemon.py -> build/lib.linux-x86_64-cpython-311/systemd/test
      copying systemd/test/test_journal.py -> build/lib.linux-x86_64-cpython-311/systemd/test
      copying systemd/test/test_login.py -> build/lib.linux-x86_64-cpython-311/systemd/test
      copying systemd/test/test_id128.py -> build/lib.linux-x86_64-cpython-311/systemd/test
      running build_ext
      building 'systemd/_journal' extension
      creating build/temp.linux-x86_64-cpython-311
      creating build/temp.linux-x86_64-cpython-311/systemd
      gcc -Wsign-compare -DNDEBUG -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -g -DOPENSSL_LOAD_CONF -fwrapv -fno-semantic-interposition -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -g -IVendor/ -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -g -IVendor/ -fPIC -DPACKAGE_VERSION=\"235\" -DLIBSYSTEMD_VERSION=254 -I/opt/yin-yang/.venv/include -I/usr/include/python3.11 -c systemd/_journal.c -o build/temp.linux-x86_64-cpython-311/systemd/_journal.o -std=c99 -Werror=implicit-function-declaration
      systemd/_journal.c:21:10: fatal error: Python.h: No such file or directory
         21 | #include <Python.h>
            |          ^~~~~~~~~~
      compilation terminated.
      error: command '/bin/gcc' failed with exit code 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for systemd-python
  Running setup.py clean for systemd-python
Failed to build systemd-python
ERROR: Could not build wheels for systemd-python, which is required to install pyproject.toml-based projects
```

### Version

the one from the repo

### How did you install Yin & Yang?

Git clone + running install script

### What desktop environments are you seeing the problem on?

KDE

### Which plugin causes the issue?

None

**Comment by l0drex** [comment]
Try to install systemd-python from your systems package manager.

**Comment by DSWNG** [comment]
I've had the same problem and I was able to solve it.
Not a bug, you just need to install some dependencies first.

I've started with python-systemd. I've found in in [RPM](https://software.opensuse.org/download/package?package=python-systemd&project=security).

Then I've installed pyside6
`sudo zypper install python3-pyside6`

and python3-devel
`sudo zypper install python3-devel`

After all the above, Yin-Yang installed successfully.
